CHANGELOG.md

```diff
@@ -14,6 +15,46 @@
 
 ## [Unreleased][]
 
+## [2.0.0][]
+
+This release involves a substantial rewrite and several breaking changes from
```

Comment:
I wonder if it would be clearer to put this in to release notes than repeat it in the changelog. I'm not sure how projects usually handle changes that only shipped in prereleases.

Reply:
You definitely need "how to upgrade" bits in the Readme - they're gonna appear on npmjs.com etc.

Reply:
I think what I've got in there looks pretty good!